pkg/network/connection.go

```diff
@@ -642,6 +702,9 @@ func (c *connection) startWriteLoop() {
 				c.Close(api.NoFlush, api.LocalClose)
 			}
 
+			if c.network == "udp" && strings.Contains(err.Error(), "connection refused") {
```

Comment:
other write errs not close connection, beacause readbuffer may have unread data, wait for readloop close connection